content/docs/setup/kubernetes/install/kubernetes/index.md

```diff
@@ -69,7 +69,7 @@ Choose this variant for:
 * Applications that use
   [liveness and readiness probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/),
 * Headless services, or
-* `StatefulSets`
+* StatefulSets
```

Comment:
That should actually just be 'stateful sets'

Two words, no caps.

Reply:
It is also a k8s concept - see https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/

Reply:
should I add this to the ignore list?

Reply:
Since it is a K8s object/concept, then it should be in back-ticks. That is what that markup indicate: https://istio.io/about/contribute/style-guide/#use-code-style-for-object-field-names

Reply:
reading this -- https://istio.io/about/contribute/style-guide/#address-the-reader-as-you

I don't think this is common to highlight in red for `Deployment` in this example in k8s docs.